pkg/config/vault.go

```diff
@@ -119,6 +119,8 @@ func GetVaultClientFromConfig(config map[string]interface{}, creds VaultCredenti
 	}
 	if err != nil {
 		log.Entry().Info("  failed")
```

Comment:
I'll try this out, logging still might be changed after testing.

Reply:
It's done now, but depends on [this HTTP pkg fix](https://github.com/SAP/jenkins-library/pull/5040), otherwise there is some HTTP logging that messes the structure up.